lib/money/money_parser.rb

```diff
@@ -7,6 +7,7 @@ def self.parse(input)
   end
   
   def parse(input)
+    input = input.to_s if !input.is_a?(String)
     Money.new(extract_money(input))
```

Comment:
I think it makes sense to do a `Money.new(extract_money(input.to_s))` here and remove the line above.